### DISCUSSION TITLE
more academic tools ( e.g., google scholar, consensus ) are needed in dify

### DISCUSSION THREAD

**Discussion by francis-ch** [langgenius/dify#6526] [Help]
Dify's plugin functionality, while promising, doesn't yet match Coze's, particularly regarding academic tools. This limitation hinders Dify's utility in research, as the agent often lacks access to relevant literature, leading to inaccurate outputs. Expanding Dify's range of academic plugins would significantly enhance its value within the academic community and broaden its overall impact.

**Comment by appwcn** [comment]
need it badly. UP
